packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -907,9 +911,24 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         private _storage?: IDocumentStorageService,
     ) {
         super();
-
         this.baseSummaryMessage = metadata?.message;
 
+        // If this is a new container, we initialize these values. If it's an existing container,
+        // these values would stay undefined
```

Comment:
This comment is incorrect. For existing container, these values are not always undefined. They are initialized from the metadata blob, if they exist in it.